components.css

```diff
@@ -1,9 +1,22 @@
-@import url("./CSS/navbar.css");
-@import url("./CSS/login.css");
+@import url("./components/navbar.css");
+@import url("./pages/login/login.css");
+@import url("./pages/products/products.css");
+@import url("./components/filters.css");
 
 :root{
     --lightergrey: #f8f8f8;
 }
 body{
     margin: 0;
+}
+.item-img{
+    width: 250px;
+    height: 300px;
+}
+.container-card{
+    width: 250px;
```

Comment:
use rems instead of px

Reply:
done [0277212](https://github.com/shivanigangadharan/e-mart/pull/3/commits/0277212dc502ea55b39a7b6fef0c44f9933cd88f)

Reply:
great